test/ownable.js

```diff
@@ -0,0 +1,49 @@
+"use strict";
+
+const Ownable = artifacts.require("Ownable");
+
+contract('Ownable', async (accounts) => {
+  let owner = accounts[0];
+  let other = accounts[1]
+  
+  let ownable = null;
+
+  beforeEach("create ownable before each", async () => {
+    ownable = await Ownable.new({from: owner});
+  })
+
+  it("should create ownable", async () => {
+    assert.isOk(ownable);
+  });
+
+  it("should saving the owner", async () => {
+    assert.equal(await ownable.owner.call(), owner);
+  });
+
+  it("should transferOwnership when sendingd from the owner", async () => {
+    await ownable.transferOwnership.sendTransaction(other, {from: owner});
+    
+    assert.equal(await ownable.owner.call(), other);
+  });
+
+  it("should transferOwnership when sending not from the owner", async () => {
+    try {
+      await ownable.transferOwnership.sendTransaction(0, {from: other});
```

Comment:
I think it would be good to use some address instead of `0`, otherwise we can't be sure if the transaction fails because it was sent to `0` address or because it was sent not by the owner.